src/psyclone/tests/conftest.py

```diff
@@ -92,10 +80,15 @@ def have_graphviz():
 
 
 @pytest.fixture(scope="session", autouse=True)
-def infra_compile(tmpdir_factory):
-    '''A per-session initialisation function that makes sure that the
-    infrastructure files for the dynamo0p3 and gocean1p0 APIs are compiled.
+def infra_compile(tmpdir_factory, request):
+    '''A per-session initialisation function that sets the compilation flags
+#    in the Compile class based on command line options for --compile,
```

Comment:
Why the comment lines here? Please fix...

Reply:
No idea ... done :)